### DISCUSSION TITLE
ML Sign off for R5.0.1

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Ed/Community#198] [General Discussion]
@vijiurs @Preethi-Sl As we have not received sign off for R5.0.1 on the planned date which was on 27th Sept 2022.Please confirm  there are no planned items for this release.

CC @sharathkashyap @luckynvdu @chitranshu-keshav @ShailandraOjha

**Comment by sharathkashyap** [comment]
@anunagaraj Locking this conversation from further discussion
